event-triggers.md

```diff
@@ -8,7 +8,7 @@ Trigger webhooks on database events using Hasura GraphQL Engine's event triggers
 
 * **Build reactive & async apps/features**: Trigger asynchronous serverless or cloud functions to reduce infrastructure costs and simplify DevOps for developers.
 
-* **Atomic & Reliable**: Using native Postgres triggers, every relevant action on the database is captured as an event. Even if Hasura is down or being updated, events will be captured and delivered as soon as possible with an *atleast-one* guarantee. You can even configure delivery policies like `max_retries` and `retry_interval`.
+* **Atomic & Reliable**: Using native Postgres triggers, every relevant action on the database is captured as an event. Even if Hasura is down or being updated, events will be captured and delivered as soon as possible with *atleast-one* guarantee. You can even configure delivery policies like `max_retries` and `retry_interval`.
```

Comment:
The correct wording should be `with an *atleast-once* guarantee`.

Reply:
Then I would make the changes as specified.